content/blog/adoptopenjdk-welcomes-dragonwell/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: AdoptOpenJDK Welcomes Dragonwell
+date: "2020-11-10T12:00:00+00:00"
+author: andreasahlenstorf
+description: With Alibaba Dragonwell, another flavour of OpenJDK is being built at and distributed by AdoptOpenJDK. Dragonwell is a TCK-certified implementation of Java for large-scale application deployments on Alibaba Cloud and elsewhere.
+tags:
+  - announcement
+---
+
+AdoptOpenJDK is happy to announce that Alibaba decided to join forces with us by building and distributing their [Dragonwell JDK](http://dragonwell-jdk.io/) on our infrastructure. For our users, this means more choice, mainly because Dragonwell offers some exciting features that are unique to Dragonwell. One example is [Wisp](https://github.com/alibaba/dragonwell8/wiki/Wisp-Documentation) that implements coroutines on the JVM.
+
+Over the next weeks, the first releases of Dragonwell 8 and 11 are going to appear on AdoptOpenJDK's website alongside our existing binary offerings. Dragonwell can be consumed like any other binary release by AdoptOpenJDK. The Dragonwell team will also introduce Dragonwell JDK and its unique features in a series of blog posts. Stay tuned!
```

Comment:
Here is a brief introduction for the available customized features in Dragonwell: 
• Multi-tenant: running multiple apps in the same instance safely
• Wisp: building the user-mode thread in Java, allowing Java developers to write code in synchronous, but gain the performance of asynchronous.
• JWarmup: priming online application for speed
• ElasticHeap:  scaling JVM's memory utilization dynamically according to the application's needs.
• JFR extension: extending more JFR events inside JVM for efficient problem diagnostics.

Reply:
If left those out. That would warrant further explanations.